src/EditorFeatures/CSharpTest/Completion/CompletionProviders/ExplicitInterfaceMemberCompletionProviderTests.cs

```diff
@@ -102,6 +102,7 @@ class Bar : IGoo
 
         [Fact, Trait(Traits.Feature, Traits.Features.Completion)]
         [WorkItem(19947, "https://github.com/dotnet/roslyn/issues/19947")]
+        [WorkItem(26595, "https://github.com/dotnet/roslyn/issues/26595")]
```

Comment:
After considering it more I think it makes more sense to expand this unit test instead of a separate test that tests a different subset of invalid characters. I wasn't able to find an example of a test with 2 WorkItem attributes. If that isn't correct I can create a separate test.

Reply:
If the same test baseline covered both bugs, the two attributes would be fine. But since part of the baseline is covering one thing and part is covering something else, splitting this in two is probably the better choice.